interpreter/src/main/scala/pl/touk/nussknacker/engine/Interpreter.scala

```diff
@@ -79,7 +79,7 @@ private class InterpreterInternal[F[_]](
           case Some(FragmentOutputVarDefinition(varName, fields)) =>
             // TODO simplify
             val withModifiedVariable = createOrUpdateVariable(ctx, varName, fields)
-            parentContext.withVariable(varName, withModifiedVariable(varName))
+            parentContext.withVariable(varName, withModifiedVariable[Any](varName))
```

Comment:
When you provide methods that are downcasting to some type given in the generic parameter, you should ensure that nobody provides `Nothing` by mistake. It is easy to do this kind of mistake, because in scala, when you don't provide a generic parameter and it can't be inferred from other parameters, scala implicitly put a `Nothing` into it. We have such a solution in `Typed.fromDetailedType`. I checked in how many places we forgot to specify the type. But after that I realize that `Context` is a part of `component-api` and I shouldn't change it now. For now, I left only a comment next to `Context` that we should change it.
TL;DR
I reverted this.